chapters/annotations.tex

```diff
@@ -188,10 +188,19 @@ \subsubsection{Plot Curves}\label{plot-curves}
 end Curve;
 \end{lstlisting}
 
-The mandatory \lstinline!x! and \lstinline!y! expressions are restricted to be component references referring to a scalar variable or \lstinline!time!.
-It is an error if \lstinline!x! or \lstinline!y! does not designate a scalar variable.
+The mandatory \lstinline!x! and \lstinline!y! expressions are restricted to be result references in the form of \lstinline[language=grammar]!result-reference! in the grammar (\cref{expressions1}), referring to a scalar variable (or a derivative thereof) or \lstinline!time!.
+It is an error if \lstinline!x! or \lstinline!y! does not designate a scalar result.
+If \lstinline!x! or \lstinline!y! is a derivative, \lstinline!der(v, $n$)!, then $n$ must not exceed the maximum amount of differentiation applied to \lstinline!v! in the model.
 When the \lstinline!unit! of an \lstinline!Axis! is non-empty, it is an error if the unit of the corresponding \lstinline!x! or \lstinline!y! expression (i.e., a variable's \lstinline!unit!, or second for \lstinline!time!) is incompatible with the axis unit.
 
+The set of available result variable and derivative trajectories in a simulation result is tool-specific, but it is recommended to include at least the ones being used in figures.
+It is not standardized how a tool should handle missing results for valid result references.
```

Comment:
```suggestion
Missing results for valid result references, should generate some form of diagnostics.
```
As indicated I don't like the ignored recommendation of line 196 and I want to specify allowed behavior in line 197. We could add that tools may suggest how to add the missing results, but it seems almost redundant.

Reply:
Agreement (except for comma and "some form").

Reply:
Done.  OK?